modules/cert-manager/cluster-issuer.tf

```diff
@@ -1,45 +1,34 @@
 locals {
-  solvers = concat(var.r53_configs.access_key_id == "" ? [] : [
-    {
-      "dns01" = {
-        "route53" = {
-          "region"       = var.r53_configs.region
-          "accessKeyID"  = var.r53_configs.access_key_id
-          "hostedZoneID" = var.r53_configs.hosted_zone_id
-          "secretAccessKeySecretRef" = {
-            "name" = "${var.clusterissuer_name}-aws-r53-secret"
-            "key"  = "secret-access-key"
-          }
-        }
-      }
-    }
-    ], [
-    {
-      "http01" = {
```

Comment:
this seems is default issuer config (no dns validation and just server direct validation of certificate), why we removed this.
there can be cases when no gcp-dns and no google-route53 needed for certificate issuer to validate certificate